pandas/tests/series/test_analytics.py

```diff
@@ -1045,6 +1030,89 @@ def test_drop_duplicates_bool(self):
         sc.drop_duplicates(keep=False, inplace=True)
         assert_series_equal(sc, tc[~expected])
 
+    @pytest.mark.parametrize('keep, expected', [
+        ('first', Series([False, False, True, False, True], name='name')),
+        ('last', Series([True, True, False, False, False], name='name')),
+        (False, Series([True, True, True, False, True], name='name'))
+    ])
+    def test_duplicated_keep(self, keep, expected):
+        s = Series(['a', 'b', 'b', 'c', 'a'], name='name')
+
+        result = s.duplicated(keep=keep)
+        tm.assert_series_equal(result, expected)
+
+    @pytest.mark.parametrize('keep, expected', [
+        ('first', Series([False, False, True, False, True])),
+        ('last', Series([True, True, False, False, False])),
+        (False, Series([True, True, True, False, True]))
+    ])
+    def test_duplicated_nan_none(self, keep, expected):
+        # np.nan and None are considered equal
```

Comment:
not considered equal here, no?

Reply:
Yes, this was a relict from the first df-tests.